casadm/cas_lib.c

```diff
@@ -576,18 +695,21 @@ int get_core_info(int fd, int cache_id, int core_id, struct kcas_core_info *info
 	}
 
 	/* internally use device special file path to describe core */
-	if (get_dev_path(info->core_path_name,
-			 info->core_path_name,
-			 sizeof(info->core_path_name))) {
-		cas_printf(LOG_WARNING, "WARNING: Can not resolve path to core "
-			"%d from cache %d. By-id path will be shown for that core.\n",
-			core_id, cache_id);
+	if (!full_path) {
+		if (get_dev_path(info->core_path_name,
+				info->core_path_name,
+				sizeof(info->core_path_name))) {
+			cas_printf(LOG_WARNING, "WARNING: Can not resolve path to core "
+				"%d from cache %d. By-id path will be shown for that core.\n",
+				core_id, cache_id);
+		}
```

Comment:
I think we can safely return failure status here

Reply:
ok